spec/chorebot_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require 'pry'
+require_relative '../chorebot'
+
+
+def member_names
+  %w(1 2 3 4 5 6 7)
+end
+
+describe 'ChoreBot' do
+  it '.assignees_for' do
+    expect(assignees_for(Date.new(2015, 6, 1), member_names)).to match_array %w(4 5 6)
+    expect(assignees_for(Date.new(2015, 6, 2), member_names)).to match_array %w(7 1 2)
+    expect(assignees_for(Date.new(2015, 6, 3), member_names)).to match_array %w(3 4 5)
+    expect(assignees_for(Date.new(2015, 6, 4), member_names)).to match_array %w(6 7 1)
+  end
```

Comment:
This could be updated to use `eq` instead of `match_array` after making `assignees_for` deterministic.